docs/user-guide/banner-section.md

```diff
@@ -0,0 +1,20 @@
+# Banner Section
+**********************
+
+The Banner Section is a [Title Section](title-section.md) without the text content to create a banner in the story through the background content.
```

Comment:
The Banner Section is similar to the [Title Section](title-section.md) and it is useful to easily create a story banner without the title text content.